Sources/Logging/Logging.swift

```diff
@@ -240,6 +244,11 @@ internal final class StdoutLogger: LogHandler {
 
     public init(label: String) {}
 
+    public func log(level: Logging.Level, message: String, metadata: Logging.Metadata, error: Error?, file: StaticString, function: StaticString, line: UInt) {
+        let prettyMetadata = !metadata.isEmpty ? self.prettify(metadata) : nil
```

Comment:
the cost is the implementations will be forced to compute the metadata representation on every call, iow it moves to the hot path